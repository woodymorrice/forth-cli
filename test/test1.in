# Test 1 is built into the test script.
# Tests each makefile target, and removes the output before testing the next.
# If any errors result, the test failed
