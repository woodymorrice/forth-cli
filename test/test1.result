20,24d19
< g++ -Wall -pedantic -std=c++17 -c -g -o out/forth-g.o src/forth.cc
< g++ -Wall -pedantic -std=c++17 -c -g -o out/datum-g.o src/datum.cc
< g++ -Wall -pedantic -std=c++17 -c -g -o out/dict-g.o src/dict.cc
< g++ -Wall -pedantic -std=c++17 -c -g -o out/stack-g.o src/stack.cc
< g++ -Wall -pedantic -std=c++17 -lm -g -o bin/forth-debug out/forth-g.o out/datum-g.o out/dict-g.o out/stack-g.o
