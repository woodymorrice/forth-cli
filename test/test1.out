g++ -Wall -pedantic -std=c++17 -c -o out/forth.o src/forth.cc
g++ -Wall -pedantic -std=c++17 -c -o out/stack.o src/stack.cc
g++ -Wall -pedantic -std=c++17 -c -o out/dict.o src/dict.cc
g++ -Wall -pedantic -std=c++17 -c -o out/datum.o src/datum.cc
g++ -Wall -pedantic -std=c++17 -c -o out/forth.o src/forth.cc
g++ -Wall -pedantic -std=c++17 -c -o out/datum.o src/datum.cc
g++ -Wall -pedantic -std=c++17 -c -o out/dict.o src/dict.cc
g++ -Wall -pedantic -std=c++17 -c -o out/stack.o src/stack.cc
g++ -Wall -pedantic -std=c++17 -lm -o bin/forth out/forth.o out/datum.o out/dict.o out/stack.o
g++ -Wall -pedantic -std=c++17 -c -o out/forth.o src/forth.cc
g++ -Wall -pedantic -std=c++17 -c -o out/datum.o src/datum.cc
g++ -Wall -pedantic -std=c++17 -c -o out/dict.o src/dict.cc
g++ -Wall -pedantic -std=c++17 -c -o out/stack.o src/stack.cc
g++ -Wall -pedantic -std=c++17 -lm -o bin/forth out/forth.o out/datum.o out/dict.o out/stack.o
g++ -Wall -pedantic -std=c++17 -c -o out/forth.o src/forth.cc
g++ -Wall -pedantic -std=c++17 -c -o out/datum.o src/datum.cc
g++ -Wall -pedantic -std=c++17 -c -o out/dict.o src/dict.cc
g++ -Wall -pedantic -std=c++17 -c -o out/stack.o src/stack.cc
g++ -Wall -pedantic -std=c++17 -lm -o bin/forth out/forth.o out/datum.o out/dict.o out/stack.o
